Android v2.1.0 post (#84)
jellyfin/jellyfin-blog: content/posts/android-v2.1.0.md

---
title: 'Android v2.1.0 - Jellyfin in your car!'
subtitle: 'Listen to your music while driving with the new Android update'
author: 'Niels van Velzen'
githubusername: 'nielsvanvelzen'
date: 2020-10-25T00:00:00-00:00
justify: 'center'
---

Jellyfin for Android version 2.1 is here with support for Android Auto.

<!--more-->

A new version of the Android app: version 2.1 has just been released! This update contains some quality of life improvements, some bugfixes and new features.
It's been only a few weeks since 2.0 but we couldn't wait to bring these new exiting changes to you. Let's start with the most awesome feature.

## Android Auto

Starting with this update we now support playing music in cars. This change was contributed by [@Spacetech](https://github.com/Spacetech) and we are excited to release it. Right now it supports browsing your music library in multiple categories: latest, albums, artists, songs and genres. It allows shuffling your albums and shows thumbnails when available.

<video controls loop autoplay muted playsinline class="inline justify" height="500">
	<source src="/images/posts/android-2-1-0/android-auto.webm" type="video/webm" />
</video>

> ___Note:__ since we do not have offline-support at this moment all music playback needs an active network connection to work. Be aware this may impact your mobile internet bundle._

Reviewer: ```suggestion
_**Note:** since we do not have offline-support at the moment all music playback needs an active network connection to work. Be aware this may cause additional charges in your mobile plan._
```

## Connectivity issues

> Some users reported issues when connecting to their server. We've made some improvements to hopefully fix these problems. When your server uses an outdated version a warning is shown. If the webui fails to load we now show a proper error allowing you to change the server address. Users with self-signed certificates should be able to use the app again now. And lastly, when your device name uses special characters they are now removed to fix the "endless loading" issue.

Reviewer: ```suggestion
Some users reported issues when connecting to their server. We've made some improvements to fix those problems:

- When your server uses an outdated version a warning is shown
- If the webui fails to load we now show a proper error allowing you to change the server address
- Users with self-signed certificates should be able to use the app again
- And lastly, when your device name includes special characters they are now removed to fix the "endless loading" issue
```

## Playback improvements

> The native video player (ExoPlayer) now supports zooming using gestures so you can remove the black bars from the video. We made some changes to which video and audio codecs are supported to prevent unnecessary transcoding. We also made some big changes to the structure of the app to fix an issue where Picture-in-Picture windows were unable to open the app when going back.

Reviewer: ```suggestion
The native video player (ExoPlayer) now supports zooming using gestures so you can remove the black bars from the video. We made some changes to which video and audio codecs are supported to prevent unnecessary transcoding. We also made some big changes to the structure of the app to fix an issue where you weren't able to return to the app after leaving a Picture-in-Picture window.
```

Author: I also changed it to "Picture-in-Picture mode" instead of "Picture-in-Picture window".

There is a new option in the settings to select which external player to use. The listed players should also report playback status back to Jellyfin to track what you watch and allow the app to resume playback. Issues with subtitles (especially external subtitles) should now be less common.

## F-Droid

> We have added the required metadata for F-Droid to our repository. We are currently working with the F-Droid team to get the app in their repository. We will make an announcement when this is ready.

Reviewer: ```suggestion
We have added the required metadata for F-Droid to our repository. We are currently working with the F-Droid team to get the app in their repository - we will post an additional announcement when this is ready.
```
